tooling/cli/src/icon.rs

```diff
@@ -41,15 +41,20 @@ pub struct Options {
   /// Default: 'icons' directory next to the tauri.conf.json file.
   #[clap(short, long)]
   output: Option<PathBuf>,
+
+  /// Extra icon sizes.
+  #[clap(short, long, use_delimiter = true)]
+  extra: Option<Vec<u32>>,
 }
 
 pub fn command(options: Options) -> Result<()> {
   let input = options.input;
   let out_dir = options.output.unwrap_or_else(|| tauri_dir().join("icons"));
+  let extra_icon_sizes = options
+    .extra
+    .unwrap_or_else(|| vec![32, 16, 24, 48, 64, 256]);
```

Comment:
```suggestion
    .unwrap_or_default();
```
sending an empty array is fine. at this point it may be worth to just send the option and handle this in `fn png` instead so that one can get rid of a potentially unused vec.